gradlew

```diff
@@ -42,11 +42,6 @@ case "`uname`" in
     ;;
 esac
 
-# For Cygwin, ensure paths are in UNIX format before anything is touched.
-if $cygwin ; then
```

Comment:
Cool. I think the test is just "Clayton makes sure this branch still builds on his machine"